biscuit-auth/src/datalog/mod.rs

```diff
@@ -707,6 +706,12 @@ impl FactSet {
         self.inner.values().fold(0, |acc, set| acc + set.len())
     }
 
+    pub fn is_empty(&self) -> bool {
+        self.inner
+            .values()
+            .fold(true, |acc, set| acc | set.is_empty())
```

Comment:
```suggestion
            .fold(true, |acc, set| acc && set.is_empty())
```

Reply:
Haha, clippy has yet another suggestion
```rust
self.inner.values().all(|set| set.is_empty())
```
but will drop it if you decide to ignore the warning.